plasmapy/formulary/relativity.py

```diff
@@ -63,7 +63,7 @@ def Lorentz_factor(V: u.m / u.s):
     inf
     """
 
-    if not np.all(np.abs(V) <= c):
+    if not np.all((np.abs(V) <= c) + (np.isnan(V))):
```

Comment:
This is purely optional, but I think this should be equivalent and might make it a little cleaner to read:
```suggestion
    if not np.all((np.abs(V) <= c) || np.isnan(V)):
```

Reply:
I hadn't realized that we could use a logical "or" operator with numpy arrays!  I only remember trying them for lists (e.g., `[True, False] | [False, False]`), which results in a `TypeError`.  The `||` is Java syntax, I think, so we'd want to use `|` here instead of `||`.

Reply:
Resolved by https://github.com/PlasmaPy/PlasmaPy/pull/1681/commits/bf92298c8d24a813890b6acbad5b7ad4f820b7de.

Reply:
Huh! I have no idea where I pulled the double pipe from. It sure wasn't Java! Thanks for spotting that, though!